blog/2021-07-15-supervisor_update.md

```diff
@@ -0,0 +1,80 @@
+
+These changes are live now on the `dev` channel for the Supervisor, so you can start testing and adjusting your tools/add-ons to make sure they will still work when your users get this.
+
+### API changes
+
+With this transition, a new base section in the Supervisor API has been added [`/backups`][supervisor_api_backups] that operates the same way as [`/snapshots`][supervisor_api_snapshots] with all the same endpoints as the old section has. But there are two key differences.
+
+- If you access `/backups` the data returned now will be `{"backups": []}` instead of `{"snapshots": []}`
+- To delete a snapshot you now have to use the `DELETE` HTTP method with the `/backups` endpoint, previously both `POST` and `DELETE` was supported.
+
+The old [`/snapshots`][supervisor_api_snapshots] endpoints are now deprecated and are scheduled for removal in Q4 of this year.
+
+### Backup structure changes
+
+For consistency, we have also changed the name of the meta file inside the backup tar from `snapshot.json` to `backup.json`, so if you have a tool that use that file, you need to adjust it to look for both.
+
+## Steaming ingress
+
+Some add-ons needs to receive large payloads from the user, for instance with uploading.
+
+Previously, there has been a limit on 16MB pr. request for add-ons running behind ingress, and this is still the default. If you need to receive bigger payloads, you can enable this by setting `ingress_stream` to `True` in the add-on configuration. When you do this the request are streamed from the client to your add-on, which have no size limit and virtually no overhead.
+
+Note that not all webservers are able to handle this by default, so you might need to adjust it.
+
+## Deprecated API endpoints
+
+Over the past years we have restructured parts of our API endpoints, but we have also kept old endpoints working. If you use any of the deprecated endpoints in your tools/add-ons you should adjust this. All deprecated endpoints are scheduled for removal in Q4 this year.
+
+Here is a list of those and their replacements:
+
+Deprecated endpoints | Replaced with
+-- | --
+`/homeassistant/*` | `/core/*`
+`/snapshots/*` | `/backups/*`
+
+In addition to this, the following are also deprecated and are also scheduled for removal in Q4 this year.
+
+- The environment variable `HASSIO_TOKEN` has been replaced with `SUPERVISOR_TOKEN`.
+- Using `X-Hassio-Key` header has been replaced with [using `Authorization` with a Bearer token][api_example].
+- Using `http://hassio/` to communicate with the Supervisor has been replaced with `http://supervisor/`.
+
+## Supervised installation
+
+Maintaining a [supervised installation][supervised_installation] is currently not the best experience. [the script][supervised_script] that most users use to install is behind what the Supervisor wants from the host. Since there are no real upgrade path for those using it users of it needs to manually adjust their installation.
```

Comment:
```suggestion
Maintaining a [supervised installation][supervised_installation] is currently not the best experience. [The script][supervised_script] that most users use to install is behind what the Supervisor wants from the host. Since there are no real upgrade paths for those using it, users of it need to manually adjust their installation.
```